src/pages/index.js

```diff
@@ -1,256 +1,231 @@
-import { config } from "../utils/constants.js";
-import "../pages/index.css";
+import {
+  config,
+  cardSelector,
+  editProfileFormElement,
+  profileEditButton,
+  profileTitleInput,
+  profileDescriptionInput,
+  profileAddButton,
+  newCardEditForm,
+  avatarForm,
+  changeAvatarImageButton,
+  removeCardForm,
+} from "../utils/constants.js";
 import Card from "../components/Card.js";
+import Api from "../components/Api.js";
 import FormValidator from "../components/FormValidator.js";
-import PopupWithForm from "../components/PopupWithForm.js";
-import PopupWithImage from "../components/PopupWithImage.js";
-import PopupWithConfirm from "../components/PopupWithConfirm.js";
 import Section from "../components/Section.js";
+import PopupWithForm from "../components/PopupWithForm.js";
+import "../pages/index.css";
 import UserInfo from "../components/UserInfo.js";
-import Api from "../components/Api.js";
-
-/////////////////////////////////////////////////// ELEMENTS ///////////////////////////////////////////////////////////////////////////
-
-//  PROFILE
-const avatarPicture = document.querySelector(".profile__avatar_edit-button");
-const profileEditButton = document.querySelector("#profile-edit-button");
-const profileEditModal = document.querySelector("#profile-edit-modal");
-const profileNameInput = document.querySelector("#profile-name-input");
-const profileDescriptionInput = document.querySelector(
-  "#profile-description-input"
-);
-const profileEditForm = profileEditModal.querySelector("#modal-form-1");
-const avatarEditModal = document.querySelector("#avatar-change-modal");
-const avatarEditForm = avatarEditModal.querySelector("#modal-form-3");
-
-// CARD TEMPLATE
-const cardTemplate =
-  document.querySelector("#card-template").content.firstElementChild;
-
-// CARD ADD
-const cardAddButton = document.querySelector("#card-add-button");
-const cardAddModal = document.querySelector("#card-add-modal");
-const cardAddForm = cardAddModal.querySelector("#modal-form-2");
-
-// UNIVERSAL SUBMIT BUTTON
-const submitButton = document.querySelector(".modal__button");
+import PopupWithImage from "../components/PopupWithImage.js";
 
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////// FORM VALIDATOR //////////////////////////////////////////////////////////////////////////////////////////
 
-// SECTION CLASS FOR CARDS
-const cardSection = new Section(
-  {
-    renderer: (item) => {
-      cardSection.addItem(makeCard(item));
-    },
-  },
-  ".cards"
-);
+// EDIT PROFILE
+const editProfileValidator = new FormValidator(config, editProfileFormElement);
+editProfileValidator.enableValidation();
+// ADD CARD
+const addCardValidator = new FormValidator(config, newCardEditForm);
+addCardValidator.enableValidation();
+// AVATAR
+const avatarFormValidator = new FormValidator(config, avatarForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object: `formValidators`.  Then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // Here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // Here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```